### PR TITLE
Feat/revised template

### PR DESCRIPTION
**Pull request by fOppenheimer**

### CODE REVIEW

**Review by ggrund-tsi — APPROVED** [review]

File: src/assets/i18n/en/translation.json [hunk 1]
```diff
@@ -111,5 +111,7 @@
   "first-positive-test-date": "First Positive Test Result",
```
Comment (PippoPalumbo) [inline src/assets/i18n/en/translation.json:111]: 02-06-2021

File: src/assets/i18n/en/translation.json [hunk 2]
```diff
@@ -111,5 +111,7 @@
...
   "recovery-country": "Country of Test",
```
Comment (PippoPalumbo) [inline src/assets/i18n/en/translation.json:112]: Italia

File: src/assets/i18n/en/translation.json [hunk 3]
```diff
@@ -111,5 +111,7 @@
...
   "cert-valid-from-to": "Certificate Valid From - To",
```
Comment (PippoPalumbo) [inline src/assets/i18n/en/translation.json:113]: 02-06-2021
30-06-2021

File: src/assets/i18n/en/translation.json [hunk 4]
```diff
@@ -111,5 +111,7 @@
...
-  "pdfSurname": "Surname(s) and forename(s)"
+  "pdfSurname": "Surname(s) and forename(s)",
+  "pdfDateOfBirth": "Date of birth",
```
Comment (PippoPalumbo) [inline src/assets/i18n/en/translation.json:115]: 13-07-1966

File: src/assets/i18n/fr/translation.json [hunk 5]
```diff
@@ -1,3 +1,5 @@
 {
-    "pdfSurname": "Nom(s) de famille et prénom(s)"
+    "pdfSurname": "Nom(s) de famille et prénom(s)",
+    "pdfDateOfBirth": "Date de naissance",
+    "pdfCi":"Identifiant unique du certificat"
```
Comment (PippoPalumbo) [inline src/assets/i18n/fr/translation.json]: 01ITAE50E47F6845472286B7AC8326E9E2D8#2

File: src/assets/i18n/en/translation.json (hunk 1 above)
Comment (PippoPalumbo) [inline src/assets/i18n/en/translation.json:111]: 02-06-2021

File: src/assets/i18n/en/translation.json [hunk 6]
```diff
@@ -111,5 +111,7 @@
   "first-positive-test-date": "First Positive Test Result",
   "recovery-country": "Country of Test",
   "cert-valid-from-to": "Certificate Valid From - To",
-  "pdfSurname": "Surname(s) and forename(s)"
```
Comment (PippoPalumbo) [inline src/assets/i18n/en/translation.json:114]: Palumbo giuseppe

File: src/assets/i18n/en/translation.json (hunk 4 above)
Comment (PippoPalumbo) [inline src/assets/i18n/en/translation.json:115]: 13-07-1966

**Pull request merged**